.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: node_js
 node_js:
-- '0.10'
+- '0.12'
+before_install:
+- phantomjs -v
 notifications:
   email: false
+sudo: false
```

Comment:
We are guaranteed < 10 sec build starts, and more consistent builds throughout the day :smiley: